app/jinja_filters.py

```diff
@@ -98,7 +102,7 @@ def format_unit_input_label(unit: str, unit_length: str = "short") -> str:
         ).replace("2 ", "")
     else:
         unit_label = units.format_unit(
-            value="",
+            value="",  # type: ignore
```

Comment:
type ignore explanation needed

Reply:
This one (and the others updated in jinja filters) is being flagged since I updated the pipenv dependencies, and are not really linked to this PR. It probably needs to be addressed properly as feels wrong passing an empty string here. I will add it as a point to fix when I raise the card to update the flask dependencies. Will add a comment for now though.

Reply:
Ah no probs at all, thanks for explaining that, will resolve 👍